frontend/packages/dev-console/integration-tests/features/addFlow/export-application.feature

```diff
@@ -0,0 +1,36 @@
+@add-flow
+Feature: Export of application from the Add page
+              As a user, I have an unmanaged application which I want to export. I'd like to be able to later add that code to git or some shared location so that I can share with others, or import into a new cluster or same cluster but different project, or be able to apply updates to an existing application.
+
+
+
+    #The operator mentioned below needs to be installed beforehand due to the issue in its installation
+        Background:
+            # Given user has installed Gitops primer Operator
```

Comment:
why is this commented? can't we install the operator

Reply:
We can, but I noticed that some errors show up in the form and don't redirect to the installed operators' page, as a result, the e2e checks fail

Reply:
@sanketpathak ^^ any suggestion/workaround here?

Reply:
This is the error that I am talking about

![Screenshot 2022-08-03 at 12 38 32 PM](https://user-images.githubusercontent.com/22490998/182546098-c1ddf28f-1114-4e14-ac6b-077a2f4642ca.png)

Reply:
We have mechanism for it here https://github.com/openshift/console/blob/64228873e86aa335169c8821294aba8eeec1ec1a/frontend/packages/dev-console/integration-tests/support/pages/functions/installOperatorOnCluster.ts#L43-L47